src/services/ubi/community.ts

```diff
@@ -1690,6 +1962,22 @@ export default class CommunityService {
                     },
                 ],
             },
+            {
+                attributes: { exclude: ['id', 'communityId'] },
+                model: this.ubiCommunityDailyState,
+                as: 'dailyState',
+                duplicating: false,
+                where: {
+                    date: yesterdayDate
+                },
+                required: false,
+            },
+            // TODO: deprecated, use dailyState
+            {
+                model: this.ubiCommunityState,
+                attributes: { exclude: ['id', 'communityId'] },
+                as: 'state',
+            },
```

Comment:
Lets really remove it

Reply:
but, today the app doesn't use this information? It is safe to remove now?

Reply:
It does. But I mean, the table needs to be removed, not the state. We need the state.

Reply:
for now, I replaced the fields `claimed`, `raised`, `beneficiaries`, and `managers`, and start to save this information in ubi_community_daily_state (like the global daily state).
but, there are some fields only in the ubi_community_state, like `claims`, `backers`, and `removedBeneficiaries`, that's the reason that I maintained these 2 tables.
I was thinking now.. maybe could be better just call the getState function instead of doing this include in the query. What do you think?

Reply:
We can do that. I'm still trying to find out how we use two APIs (once we add TheGraph). Either query from backend and return the complete results or we query on frontend.
But yes, we can use getState here instead.